parsers/ES.py

```diff
@@ -1,27 +1,10 @@
 #!/usr/bin/env python3
 
-from collections.abc import Callable
 from datetime import datetime, timedelta, timezone
+from json import loads
 from logging import Logger, getLogger
-
-# package "ree" is used to parse data from www.ree.es
-# maintained on github by @hectorespert at https://github.com/hectorespert/ree
-from ree import (
-    Ceuta,
-    ElHierro,
-    Formentera,
-    Gomera,
-    GranCanaria,
-    IberianPeninsula,
-    Ibiza,
-    LanzaroteFuerteventura,
-    LaPalma,
-    Mallorca,
-    Melilla,
-    Menorca,
-    Response,
-    Tenerife,
-)
+from typing import Dict, Optional
```

Comment:
Neither of these type imports are needed in python 3.10

We can do `dict` instead of `Dict` and `type | None` instead of `Optional[type]`.